python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -628,17 +628,17 @@ def SetCurrentLayoutDataSendOnObjectUpdateData(self, doUpdate, containername=Non
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, fireandforget=fireandforget)
 
-    def ManuallyPlacePackItem(self, packFormationComputationResult=None, inputPartIndex=None, placeLocationNames=None, placedTargetPrefix=None, dynamicGoalsGeneratorParameters=None, orderNumber=None, numLeftToPick=None, timeout=10, fireandforget=False):
+    def ManuallyPlacePackItem(self, packFormationComputationResult=None, inputPartIndex=None, placeLocationNames=None, placedTargetPrefix=None, packFormationParameters=None, orderNumber=None, numLeftToPick=None, timeout=10, fireandforget=False):
```

Comment:
```suggestion
    def ManuallyPlacePackItem(self, packFormationComputationResult=None, inputPartIndex=None, placeLocationNames=None, placedTargetPrefix=None, dynamicGoalsGeneratorParameters=_DEPRECATD, orderNumber=None, numLeftToPick=None, packFormationParameters=None, timeout=10, fireandforget=False):
```
Technically, this is how we should deprecate in the short term, but since our API is still unstable, I am OK with the breaking change as well.

Reply:
I agree, but this part of API is indeed not very stable, so I think there's no need for this.